src/classes/STG_PanelHealthCheck_CTRL.cls

```diff
@@ -620,6 +620,25 @@ public without sharing class STG_PanelHealthCheck_CTRL extends STG_Panel {
             createDR(Label.healthLabelTriggerHandlerCheck, statusError, ex.getMessage(), null);                   
         }    
     }
+
+    /**
+     * @description Compare the Trigger_Action list between two trigger handlers usnig logic that can ignore
+     * both case and the sequence of the actions in the list.
+     */
+    private static Boolean compareTriggerHandlerActions(Trigger_Handler__c handlerA, Trigger_Handler__c handlerB) {
+        List<String> listA = handlerA.Trigger_Action__c.toLowerCase().split(';');
+        List<String> listB = handlerB.Trigger_Action__c.toLowerCase().split(';');
+
+        Set<String> unionBintoA = new Set<String>();
+        unionBintoA.addAll(listA);
+        unionBintoA.addAll(listB);
+
+        // The number of elements in in unionBintoA should match the number of elements in listA and listB
+        if (listA.size() != listB.size() || unionBintoA.size() != listA.size() || unionBintoA.size() != listB.size()) {
```

Comment:
Hi @force2b, I'm finding this conditional a little hard to follow. Isn't the requirement that the values are the same so A should always be equal to B and the values should be the same not including order? Could we just create 2 sets here and attempt adding all of set B to set A and see if that returns false and then check the flip of adding all of A into B and verify that returns false as well?

Reply:
I didn't feel it was necessary to flip the merges. Here is I looked at the test and result:

*Test Scenario 1:*
* Set A: { A1, B2, C3 }
* Set B: { A1, c3, B2 }
* Merged (lowercase): { a1, b2, c2 }
* Pass: The size of all the is still the same.

*Test Scenario 2*
* Set A: { A1, B2, C3 }
* Set B: { A1, c3, D2 }
* Merged (lowercase): { a1, b2, c3, d2 }
* Fail: The size of the merged set is not the same as sets A and B

*Test Scenario 3:*
* Set A: { A1 }
* Set B: { A1, B2, C3 }
* Merged (lowercase): { a1, b2, c3 }
* Fail: The size of the merged set is not the same as set A

Reply:
Yup, for whatever reason I thinking unionAintoB was another list not a set. Thanks for clarifying!